File: zerojudge/a004/main.py
while True:
    try:
        year = int(input())
        print("閏年" if (((year%4)==0) and ((year%100)!=0) or ((year%400) ==0)) else "平年")
        # if (((year%4)==0) and ((year%100)!=0) or ((year%400) ==0)):
        #     print("閏年")
        # else:
        #     print("平年")
    except EOFError:
        break

File: zerojudge/a004/main.cpp
#include<iostream>
using namespace std;
int main(){
    int year;
    while (cin>>year){
    if ( year%400==0 or ( year%4==0 and year%100!=0 ) ) cout<<"閏年"<<endl;
    else cout<<"平年"<<endl;
    }
}